service/header_service_test.go

```diff
@@ -348,6 +349,153 @@ func TestMerkleRootConfirmations(t *testing.T) {
 	}
 }
 
+func TestLocateHeadersGetHeaders(t *testing.T) {
+	tData := setUpServices()
+
+	testCases := []struct {
+		name                 string
+		locator              []*chainhash.Hash
+		hashstop             *chainhash.Hash
+		result               []*wire.BlockHeader
+		expectedErrorMessage string
+	}{
+		{
+			name: "happy path, all block locator hashes are valid (hashStart height equals 3), hashstop is valid (height equals 4), returns block with height 4",
+			locator: []*chainhash.Hash{
+				toChainhashPtr("0000000082b5015589a3fdf2d4baff403e6f0be035a5d9742c1cae6295464449"),
```

Comment:
there is an util method to create hash from string:

```suggestion
			fixtures.HashOf("0000000082b5015589a3fdf2d4baff403e6f0be035a5d9742c1cae6295464449"),
```

maybe it's good idea to use it.
Unfortunately it doesn't handle errors inside, so it would be good if you would add panic when error is there.